src/Microsoft.ML.ImageAnalytics/ImageType.cs

```diff
@@ -14,14 +16,14 @@ public sealed class ImageType : StructuredType
         public ImageType(int height, int width)
            : base(typeof(Bitmap))
         {
-            Contracts.CheckParam(height > 0, nameof(height));
-            Contracts.CheckParam(width > 0, nameof(width));
+            Contracts.CheckParam(height > 0, nameof(height), nameof(height) + " should be a positive number");
```

Comment:
Please do not include the name of the parameter in the message... the exception already includes the name, as you see by the fact that `nameof(height)` was already passed in, literally right before you did you message change. The message should be "Must be positive."

Reply:
will do , thank u

---
In reply to: [207391662](https://github.com/dotnet/machinelearning/pull/620#discussion_r207391662) [](ancestors = 207391662)